packages/react-components/eslint-plugin-react-components/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33214&commitId=83395d7d40d952d9e7ab1c7e4d621d4ce14df7d7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+image+badge.chromium.png|1| Changed|
|Avatar Converged.badgeMask.chromium.png|3| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1139| Changed|
|Drawer.Full Overlay RTL.chromium.png|1167| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>